lib/fauna/errors.rb

```diff
@@ -43,8 +62,12 @@ def self.raise_for_status_code(request_result)
     #            :: A simple string as the message.
     def initialize(request_result)
```

Comment:
We could just not document this. Our users won't need to create these themselves.